core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -1255,6 +1282,15 @@ object KafkaConfig {
       .define(PasswordEncoderCipherAlgorithmProp, STRING, Defaults.PasswordEncoderCipherAlgorithm, LOW, PasswordEncoderCipherAlgorithmDoc)
       .define(PasswordEncoderKeyLengthProp, INT, Defaults.PasswordEncoderKeyLength, atLeast(8), LOW, PasswordEncoderKeyLengthDoc)
       .define(PasswordEncoderIterationsProp, INT, Defaults.PasswordEncoderIterations, atLeast(1024), LOW, PasswordEncoderIterationsDoc)
+
+      /** ********* Raft Quorum Configuration *********/
+      .define(QuorumVotersProp, LIST, Defaults.QuorumVoters, HIGH, QuorumVotersDoc)
```

Comment:
I'm considering if we should define these as internal configurations for now until we have a plan for how to expose them when KIP-500 is initially released in alpha.

Reply:
That makes sense. I didn't know about the internal definition. Changed.